packages/web-react/src/components/FileUploader/demo/FileUploaderAttachment.tsx

```diff
@@ -30,6 +31,15 @@ const FileUploaderAttachment = () => {
             alert('Edit');
           }}
         />
+        <FileUploaderAttachmentComponent
+          hasImagePreview
+          name="test"
+          file={new File([byteArrayPortrait], 'test1.png', { type: 'image/png', lastModified: 123456789 })}
+          id="2"
+          label="Contained image"
+          onDismiss={onDismiss}
+          imageObjectFit="contain"
```

Comment:
It seems we do not have a unit test for this property, or do we?

Reply:
test added